### ISSUE TITLE
Add more detailed information

### ISSUE THREAD

**Issue by fkling** [fkling/jsbasics#2]
Each slide should have an "In depth" section which goes into more detail of the topic (e.g. more details about numbers or arrays).

**Comment by vamsipavanmahesh** [comment]
Basically you opened issue in your own repo :) cool man. What do you have in mind regarding design? Is like there should be a button which would expand a panel?

**Comment by fkling** [comment]
Yeah, I was thinking about having a section that can be expanded. However, I may just have the section permanently visible after the basic introduction for now.
